docs/quickstart/cross-chain-transfers.md

```diff
@@ -1,11 +1,9 @@
-# Transfers AVAX Tokens Between Chains
+# Cross Chain Transfers
 
 ## Introduction
 
 This article shows how to transfer AVAX tokens programmatically between any two chains of the Primary Network.
 
-If you are looking for how to transfter AVAX tokens using the web wallet, please check out [this article](https://support.avax.network/en/articles/6169872-how-to-make-a-cross-chain-transfer-in-the-avalanche-wallet).
```

Comment:
@Julian-dev28 if looks that you overwrote my changes. Please revert them.

Reply:
Resolved
https://github.com/ava-labs/avalanche-docs/pull/782/commits/b4ca846c0b718df3839a23ee05362b7e176b8ab1